src/js/legacy/menu.js

```diff
@@ -1,28 +1,32 @@
-$(document).ready(function() {
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
If for some reason this script is run after the content loaded event, this listener won't run. This is different from the jQuery ready function, I believe, which will run the function if the doc is in the ready state already.

I don't think that matters right now, but it could in the future if we load bundles with defer or async.